docs/guides/docker.md

```diff
@@ -0,0 +1,80 @@
+
+## Dockerise a Waku app
+Bundle your Waku app into a Docker container for portability and easy deployment in various environments.
+
+### Prerequisites
+Make sure you have Docker and Node.js installed on your machine. I used `yarn` as my packager manager, customise setup to your manager. We will prepare the image and container in 3 easy steps for reproducible results. Create 3 files in your folder root: `Dockerfile`, `.dockerignore`, `docker-compose.yml`.
```

Comment:
> I used `yarn` as my packager manager

It's a weird phrase as a guide. Something like "In this guide, we use `yarn`..."

Reply:
It should be changed to use `npm` instead for consistency.

Reply:
The sentence has been removed.
All mentions of `yarn` were replaced by `npm`